Libraries/Components/Picker/__tests__/__snapshots__/Picker-test.js.snap

```diff
@@ -64,7 +64,6 @@ exports[`<Picker /> should render as expected: should deep render when not mocke
 
 exports[`<Picker /> should render as expected: should shallow render as <Picker /> when mocked 1`] = `
 <Picker
-  mode="dialog"
```

Comment:
Oh interesting, I wouldn't have thought these snapshots would change. Shouldn't we still be setting the prop?

Reply:
Because snap only print props of passing to Picker.
After removing defaultProps，it doesn't exist in props.

Reply:
It should still exist in the props we pass because we're setting a default value via the destructured assignment
`{mode = MODE_DIALOG... `

Reply:
No, it is a single var rather than a part of props by destructured assignment.

Reply:
I think this change is expected. This test is when the component is mocked, so what shows up in the snapshot is exactly what props were passed to Picker, ignoring anything internal to picker. 

The change in the test below this kinda surprises me because the title of the test says that isn't mocked. I bet something is wrong in the test there.

Reply:
`const { mode } = this.props`

By destructured assignment，declare a `mode` var from `mode` prop of `this.props`，The default value is belong to `mode` var rather than `mode` prop of `this.props`

Reply:
I observed the original snapshot and found that the results of mock and non-mock are the same, both showing the props passed to Picker.

Reply:
Ah I see, missed the fact that it was mocked. @kerm1it did you want to look into why the second snapshot isn't passing `mode`? If not, I can follow up

Reply:
I have explain the reason in https://github.com/facebook/react-native/pull/31644#discussion_r644238493 from my opinion.

If you can follow up, I will be happy.